### PR TITLE
Small fixes5087

### PR DESCRIPTION
**Pull request by SamirTouira**
Related to the movie front improvements list: #5087

### CODE REVIEW

File: libs/movie/src/lib/movie/form/media-videos/media-videos.component.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export class MovieFormMediaVideosComponent implements OnInit, OnDestroy {
   allowedFilesTypes = allowedFiles.video.mime;
   allowedFilesExtensions =  allowedFiles.video.extension;
 
-  videoTypes = Object.keys(hostedVideoTypes);
+ public videoTypes = hostedVideoTypes;
```
Comment (Mussuranae) [inline libs/movie/src/lib/movie/form/media-videos/media-videos.component.ts:26]: I think you don't need it anymore

File: libs/movie/src/lib/movie/form/media-videos/media-videos.module.ts [hunk 2]
```diff
@@ -31,6 +33,8 @@ import { MatSelectModule } from '@angular/material/select';
     MaxLengthModule,
     FileUploaderModule,
     FileNameModule,
+    ToLabelModule,
```
Comment (Mussuranae) [inline libs/movie/src/lib/movie/form/media-videos/media-videos.module.ts:36]: Also with the use of the static-select, you don't need to import the ToLabel pipe anymore ;)

**Review by Mussuranae — APPROVED** [review]

File: libs/movie/src/lib/movie/form/media-videos/media-videos.component.ts (hunk 1 above)
Comment (SamirTouira) [reply libs/movie/src/lib/movie/form/media-videos/media-videos.component.ts]: I guess you mean about the 'public'?

Comment (Mussuranae) [reply libs/movie/src/lib/movie/form/media-videos/media-videos.component.ts]: No no the entire line, you can remove it, we needed it for a classic select, but with the static-select component, we don't need it anymore.

File: libs/movie/src/lib/movie/form/media-videos/media-videos.component.ts [hunk 3]
```diff
@@ -22,7 +22,8 @@ export class MovieFormMediaVideosComponent implements OnInit, OnDestroy {
   form = this.shell.getForm('movie');
   movieId = this.route.snapshot.params.movieId;
 
-  videoTypes = Object.keys(hostedVideoTypes);
+  allowedFilesTypes = allowedFiles.video.mime;
+  allowedFilesExtensions =  allowedFiles.video.extension;
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/media-videos/media-videos.component.ts:26]: vars are not used

File: libs/movie/src/lib/movie/form/media-videos/media-videos.component.html [hunk 4]
```diff
@@ -42,14 +42,9 @@ <h3>Other Videos</h3>
                 <mat-label>Title</mat-label>
                 <input matInput type="text" placeholder="Title of the video" [formControl]="activeForm.get('title')" />
               </mat-form-field>
-              <mat-form-field appearance="outline" fxFlex>
+              <static-select scope="hostedVideoTypes" [control]="activeForm.get('type')" fxFlex>
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/media-videos/media-videos.component.html:45]: dont' remove `test-id="video-type"` (used for e2e tests)

File: libs/movie/src/lib/movie/form/media-videos/media-videos.component.ts (hunk 3 above)
Comment (SamirTouira) [reply libs/movie/src/lib/movie/form/media-videos/media-videos.component.ts:26]: yes it was due to the conflicts, I had accepted the both changes. can I remove it?

File: libs/movie/src/lib/movie/form/sales-pitch/sales-pitch.component.html [hunk 5]
```diff
@@ -15,9 +15,9 @@ <h3>Sales Pitch</h3>
     </section>
 
     <section id="sales-video-pitch" fxLayout="column">
-      <h3>Sales Video Pitch</h3>
-      <file-uploader [form]="salesPitch" [meta]="['movies', 'salesPitch', movieId]" accept="video" listenToChanges>
-        <h3 title>Sales Video Pitch</h3>
+      <h3>Sales Pitch Video</h3>
+      <file-uploader [form]="salesPitch.get('ref')" [storagePath]="getPath()" allowedFileType="video" >
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/sales-pitch/sales-pitch.component.html:19]: `listenToChanges` not needed ?

File: libs/movie/src/lib/movie/form/media-videos/media-videos.component.html (hunk 4 above)
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/media-videos/media-videos.component.html:45]: var `hostedVideoTypes` is not declared in ts

File: libs/movie/src/lib/movie/form/media-videos/media-videos.component.ts (hunk 3 above)
Comment (phpgeekfr) [reply libs/movie/src/lib/movie/form/media-videos/media-videos.component.ts]: what is the use of keeping unused vars ?

**Pull request closed without merging**